packages/azure-kusto-data/src/index.ts

```diff
@@ -26,4 +26,5 @@ export {
     kustoTrustedEndpoints,
     MatchRule,
     TimeUtils,
+    CloudInfo
```

Comment:
That's good for now, but will probably make a conflict when they split it into import & import type. Doesn't really matter for now, just remember for later.

Reply:
i dont think they will